### PR TITLE
Feature/gha poc

### PR DESCRIPTION
**Pull request by 783967**
GHA Yml file addd

### CODE REVIEW

**Review by subhamayghosh — APPROVED** [review]
approved

File: src/test/java/swaglabs/tests/validateHomePageTest.java [hunk 1]
```diff
@@ -1,9 +1,9 @@
-package swaglabs.ai.tests;
+package swaglabs.tests;
 
 import org.testng.annotations.Test;
 
-import swaglabs.TestComponents.DriverFactory;
```
Comment (Priyanka2245968) [inline src/test/java/swaglabs/tests/validateHomePageTest.java:5]: Please add more information here, and fix this typo.

**Review by Priyanka2245968 — CHANGES_REQUESTED** [review]
This is close to perfect! Please address the suggested inline change.

File: src/test/java/swaglabs/tests/validateHomePageTest.java (hunk 1 above)
Comment (Priyanka2245968) [inline src/test/java/swaglabs/tests/validateHomePageTest.java:5]: Please add more information here, and fix this typo.

**Review by Priyanka2245968 — CHANGES_REQUESTED** [review]
This is close to perfect! Please address the suggested inline change.

File: src/test/java/swaglabs/tests/validateHomePageTest.java [hunk 2]
```diff
@@ -1,9 +1,9 @@
-package swaglabs.ai.tests;
+package swaglabs.tests;
 
 import org.testng.annotations.Test;
 
-import swaglabs.TestComponents.DriverFactory;
-import swaglabs.stepDefinitions.HomePageSteps;
```
Comment (Priyanka2245968) [inline src/test/java/swaglabs/tests/validateHomePageTest.java:6]: Please add more information here, and fix this typo.

**Review by Priyanka2245968 — CHANGES_REQUESTED** [review]
This is close to perfect! Please address the suggested inline change.

File: src/main/java/swaglabs/steps/HomePageSteps.java [hunk 3]
```diff
@@ -1,10 +1,10 @@
-package swaglabs.stepDefinitions;
+package swaglabs.steps;
 
 import org.apache.log4j.Logger;
 import org.openqa.selenium.WebDriver;
 
-import swaglabs.TestComponents.BasePage;
-import swaglabs.TestComponents.DriverFactory;
+import swaglabs.components.BasePage;
```
Comment (Priyanka2245968) [inline src/main/java/swaglabs/steps/HomePageSteps.java:6]: abcd

**Pull request merged**